component/src/main/java/io/siddhi/extension/io/jms/source/JMSSource.java

```diff
@@ -121,24 +121,59 @@
                         defaultValue = "default")
         },
         examples = {
-                @Example(description = "This example shows how to connect to an ActiveMQ topic and "
-                        + "receive messages.",
-                        syntax = "@source(type='jms', @map(type='json'), "
+                @Example(syntax = "@source(type='jms', @map(type='json'), "
                                 + "factory.initial='org.apache.activemq.jndi.ActiveMQInitialContextFactory', "
                                 + "provider.url='tcp://localhost:61616',"
                                 + "destination='DAS_JMS_TEST', "
                                 + "connection.factory.type='topic',"
                                 + "connection.factory.jndi.name='TopicConnectionFactory'"
                                 + ")\n" +
-                                "define stream inputStream (name string, age int, country string);"),
-                @Example(description = "This example shows how to connect to an ActiveMQ queue and "
-                        + "receive messages. Note that we are not providing properties like connection factory type",
-                        syntax = "@source(type='jms', @map(type='json'), "
+                                "define stream inputStream (name string, age int, country string);",
+                        description = "This example shows how to connect to an ActiveMQ topic and receive messages."),
+                @Example(syntax = "@source(type='jms', @map(type='json'), "
                                 + "factory.initial='org.apache.activemq.jndi.ActiveMQInitialContextFactory', "
                                 + "provider.url='tcp://localhost:61616',"
                                 + "destination='DAS_JMS_TEST' "
                                 + ")\n" +
-                                "define stream inputStream (name string, age int, country string);")
+                                "define stream inputStream (name string, age int, country string);",
+                        description = "This example shows how to connect to an ActiveMQ queue and receive messages." +
+                                " Note that we are not providing properties like connection factory ."),
+                @Example(syntax = "@source(type = 'jms', destination = 'Q'," +
+                                "factory.initial = 'com.tibco.tibjms.naming.TibjmsInitialContextFactory'," +
+                                "provider.url = 'tibjmsnaming://localhost:7222'," +
+                                "connection.factory.jndi.name = 'QueueConnectionFactory'," +
+                                "connection.factory.type = 'queue', java.naming.security.principal = 'USER_NAME'," +
+                                "java.naming.security.credentials = 'PASSWORD', transport.jms.UserName = 'EMSUSER'," +
```

Comment:
we don't need tarnsport.jms.username property.  Also did we test whether java.naming.security.principal is accepted by the source?